repository/Seaside-Core.package/WAComboResponse.class/instance/close.st

```diff
@@ -3,5 +3,14 @@ close
 	"Send a zero-sized chunk to end the data transfer."
 
 	closed ifTrue: [ self error: 'Response is closed' ].
-	externalStream nextPutAll: '0'; crlf; crlf; flush.
+	externalStream
+		"zero length chunk"
+		nextPut: 0;
+		"crlf"
+		nextPut: 13;
```

Comment:
I don't know how portable this is but one could also consider

    Character cr greaseInteger

Reply:
Yes. I wanted this to be fast though.

Reply:
Fair enough